docs/modules/automation-testing/pages/scripting/migrate-to-java-client-9_2_2.adoc

```diff
@@ -0,0 +1,81 @@
+= Migrate Java test to Java Appium Client 9.2.2 or above
+:navtitle: Migrate Java test to Java Appium Client 9.2.2 or above
+
+
+Java Appium Client (`java-client`) 9.2.2 or above introduce several changes that cause issues with automation test:
```

Comment:
introduces several changes that may cause the following issues with automation testing.